app/app/BaseUser.php

```diff
@@ -174,12 +205,20 @@ public function getDistanceTo($longitude, $latitude)
 			$longitude, $latitude,
 			$this->getLongitude(), $this->getLatitude());
 	}
-	
-	public static function signIn($email) 
+
+	public static function signIn($email)
 	{
-		Session::put('email',$email);
+				Session::put('email',$email);
```

Comment:
Fix indentation problem in signIn.  It looks like your tabs and spaces are mixed together.